pytest-embedded-serial-esp/pytest_embedded_serial_esp/serial.py

```diff
@@ -14,6 +15,17 @@
 from pytest_embedded_serial.dut import Serial
 
 
+def is_port_mac_verified(port: str, port_mac: str) -> bool:
+    try:
+        output = subprocess.check_output(['esptool.py', '--port', port, 'read_mac'], stderr=subprocess.STDOUT).decode()
+    except subprocess.CalledProcessError:
+        return False
+    if f'MAC: {port_mac.lower()}' in output:
+        return True
+    else:
+        return False
```

Comment:
please use `live_print_call(cmd, msg_queue=msg_queue)` and `dut.expect(f'MAC: {port_mac.lower()}')` inside the `EspSerial` class

The benefit to use `live_print_call` is that:
1. it would print the stdout to the terminal (most important)
2. it would check the return code automatically
3. you could do some expect on the output